server/app.py

```diff
@@ -9,3 +11,70 @@
 app.register_blueprint(home_handler)
 app.register_blueprint(ping_handler)
 
+# Database variables
+user = 'postgres'
+pw ='put-your-password-here'
+url = 'localhost:5432'
+db_name = 'team-bluejay'
+
+DB_URL = 'postgresql+psycopg2://{user}:{pw}@{url}/{db}'.format(user=user,pw=pw,url=url,db=db_name)
+
+app.config['SQLALCHEMY_DATABASE_URI'] = DB_URL
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+db = SQLAlchemy(app)
+ma = Marshmallow(app)
+
+# Basic User Model from which Customers and Chefs inherit from
+class User(db.Model):
```

Comment:
Is there a way we can get these classes to exist in a directory like (server/models/user.py)

Reply:
we can store the database variables in the .env then access it by 
```suggestion
import os
os.environ["DB_URL"]
```